corehq/apps/app_manager/suite_xml/post_process/remote_requests.py

```diff
@@ -270,6 +273,14 @@ def _remote_request_query_datums(self):
                     ref=f"'{self.module.search_config.custom_related_case_property}'",
                 )
             )
+        if (module_uses_include_related_cases(self.module)
+                and toggles.USH_CASE_CLAIM_UPDATES.enabled(self.app.domain)):
```

Comment:
I don't think the toggle is necessary here. There's no issue with backwards compatibility so as long as it's configured in the app then we can add it to the query

Reply:
Removed-  [70b1ed9](https://github.com/dimagi/commcare-hq/pull/32587/commits/70b1ed9cf58d255c14e8062b3874965738fb09ec)